.travis.yml

```diff
@@ -2,7 +2,6 @@ language: python
 dist: focal
 
 python:
-  - "2.7"
```

Comment:
Once this is implemented, we may want to go back and update the wiki on database migration: https://wiki.uchicago.edu/pages/viewpage.action?spaceKey=CDIS&title=Database+migration

Reply:
Thank you! Updated the wiki to mention gdcdatamodel and gdc-ng-model need python 3.5+